spec/game_runner_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "spec_helper"
+require_relative "../game_runner"
+
+RSpec.describe "GameRunner" do
+  describe "#run" do
+    it "displays an empty board" do
+      BOARD = " __ __ __\n|  |  |  |\n|__|__|__|\n|  |  |  |\n|__|__|__|\n|  |  |  |\n|__|__|__|\n"
+      io = instantiate_stdout
+
+      GameRunner.run
+
+      io.rewind
+      expect(io.read).to eq(BOARD)
+    end
+  end
+
+  def instantiate_stdout
+    io = StringIO.new("")
+    $stdout = io
```

Comment:
I'd expect overwriting this global to have an effect on other things that print. Does it affect RSpec's output?

The instructions mention this, to avoid issues that can crop up from fiddling with it:

* Dependency inject. Pass in stdin and stdout so you can pass in an alternative.

Reply:
I didn't quite understand how to do this. I took a stab at it here: 75edfe00ad7ca329f9038804d26939b97830a76d. I'm not sure how the method can return the output to use in my test.

Reply:
Your stab is on the intended path. To test the output, you could capture the StringIO in a local variable, then make assertions on it. Something like

```ruby
output = StringIO.new
GameRunner.run(output)
expect(output.string).to ...
```

Reply:
Thank you!